fuzz/fuzz_targets/decode.rs

```diff
@@ -0,0 +1,70 @@
+#![no_main]
+use std::task::{Context, Poll};
+
+use libfuzzer_sys::fuzz_target;
+use std::io;
+use futures::stream::StreamExt;
+use std::pin::Pin;
+extern crate tokio_websockets;
+use tokio::io::{AsyncRead, AsyncWrite, ReadBuf};
```

Comment:
Could you please format these using the main rustfmt.toml? I assume symlinking it in the fuzz subdirectory should be sufficient.

Reply:
Done; no symlink required as `rustfmt.toml` in any parent directory works (https://github.com/rust-lang/rustfmt/blob/182a20378b3a844e3677ff93d96f8ac70b9f009a/Configurations.md?plain=1#L3)